ih/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16.13.0-alpine AS dev
+ENV NODE_ENV development
+
+WORKDIR /app
+
+COPY package.json .
+RUN npm install
+#assuming we use yarn one day we'll need this
+#COPY yarn.lock . 
+
+COPY . . 
```

Comment:
whats this do?

Reply:
that will copy everything in the directory into the working directory inside our container because the Dockerfile is located in the root of the project